src/Lykke.Job.PayTransactionHandler.Services/Transactions/TransactionsScanService.cs

```diff
@@ -95,8 +94,7 @@ public class TransactionsScanService : IScanService
                             }
                             catch (Exception ex)
                             {
-                                await _log.WriteErrorAsync(nameof(TransactionsScanService), nameof(ExecuteAsync),
-                                    updateRequest?.ToJson(), ex);
+                                _log.Error(ex, context: updateRequest);
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.